src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx

```diff
@@ -10,9 +10,8 @@ interface FairHeaderImageProps {
 export const FairHeaderImage: React.FC<FairHeaderImageProps> = ({
   fair: { image },
 }) => {
-  if (!image) return null
+  if (!image?.url) return null
 
-  // @ts-expect-error STRICT_NULL_CHECK
   return <FullBleedHeader src={image.url} />
```

Comment:
I think that this option will be more readable

```javascript
if (image?.url) {
 return <FullBleedHeader src={image.url} />
}
return null
```

Reply:
Funnily enough I think the opposite is the case — makes more sense to early return nulls and the default case is the existy thing. (It doesn't really matter to me either way though)